docs/fields.md

```diff
@@ -2001,6 +2037,38 @@ v3.6 and after: StopStrategy defines if the CronWorkflow should stop scheduling
 |:----------:|:----------:|---------------|
 |`condition`|`string`|v3.6 and after: Condition is an expression that stops scheduling workflows when true. Use the variables `failed` or `succeeded` to access the number of failed or successful child workflows.|
 
+## Event
+
+_No description available_
```

Comment:
It would be good to go through and add descriptions at some point, but this isn't the only place where we haven't got them.